.github/actions/javascript/bumpVersion/bumpVersion.ts

```diff
@@ -47,8 +48,12 @@ if (!semanticVersionLevel || !Object.keys(versionUpdater.SEMANTIC_VERSION_LEVELS
     console.log(`Invalid input for 'SEMVER_LEVEL': ${semanticVersionLevel}`, `Defaulting to: ${semanticVersionLevel}`);
 }
 
-const {version: previousVersion} = JSON.parse(fs.readFileSync('./package.json').toString());
-const newVersion = versionUpdater.incrementVersion(previousVersion, semanticVersionLevel);
+const {version: previousVersion}: PackageJson = JSON.parse(fs.readFileSync('./package.json').toString());
+if (!previousVersion) {
+    core.setFailed('Error: Could not read package.json');
```

Comment:
```suggestion
    core.setFailed('Error: Could not read package.json');
    return;
```

Alternative: put a return here